libparsers/tests/xml/dynamic_background.cpp

```diff
@@ -0,0 +1,152 @@
+#include "parsers/xml/dynamic_background.hpp"
+#include "utils.hpp"
+#include <doctest.h>
+#include <iostream>
+
+using namespace parsers::xml;
```

Comment:
Eliminate this using namespace. You may use a type alias (`namespace newname = parsers::oldname`) but don't pull in the whole thing.